.chloggen/dotnet-instrumentation-command-flag.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Please set it as an enhancement.

Reply:
Why should it be an enhancement? We're removing a beta feature flag that users might have set. That's a breaking change imo, and that's how we've treated similar changes in the past.